### ISSUE TITLE
Launch Daemons are detected as [loaded], even when the daemon is uninstalled

### ISSUE THREAD

**Issue by ened** [etresoft/EtreCheck#4]
etreCheck finds this:

```
Launch Daemons:
    [loaded]    com.adobe.SwitchBoard.plist [Support]
```

And when I check the plist file, it contains this:

```
	<key>ProgramArguments</key>
	<array>
		<string>/Library/Application Support/Adobe/SwitchBoard/SwitchBoard.app/Contents/MacOS/launch.switchboard</string>
```

However, the program in question does not exist:

```
$ ls /Library/Application\ Support/Adobe/SwitchBoard/SwitchBoard.app/Contents/MacOS/launch.switchboard
ls: /Library/Application Support/Adobe/SwitchBoard/SwitchBoard.app/Contents/MacOS/launch.switchboard: No such file or directory
```

It would be a nice addition to show a warning/error next to these items and inform the user that the .plist is invalid.

**Comment by ened** [comment]
Same for symlinked .plists whose link destination does not exist anymore.

**Comment by etresoft** [comment]
It looks like I only check for executable not found for Apple tasks. That's not right. I don't think I've ever though about symlinks. Thanks!

**Comment by etresoft** [comment]
Not so fast! I went to make this fix and remembered that I had already made this fix - twice. Both times I eventually backed it out. There are a number of popular 3rd party tools that are installed with really bad permissions. EtreCheck would then report a problem with them when they were actually running fine. The problem is when software designed to run as root gets installed into a directory where only root has read and execute permissions. EtreCheck can't determine the validity of that executable and would report an error. I got tired of trying to explain that to people who didn't understand. Sometimes, EtreCheck just has to ignore some problems that it can't do anything about if they really aren't a problem. I'm afraid this is one of those cases. If you have any better idea, I would love to hear it. But I don't want to add this feature and back it out for a 3rd time.
